src/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Compression App</h1>
+    <br/>
+    <form action="http://localhost:5700/compress">
```

Comment:
The form's `action` attribute should specify the method as `POST` to match the server's expected method for the `/compress` endpoint. Add `method="post"` to the form tag.